heliopy/data/omni.py

```diff
@@ -0,0 +1,149 @@
+             'Long. Angle of Aver. Field Vector', 'Bx GSE, GSM', 'By GSE',
+             'Bz GSE', 'By GSM', 'Bz GSM', 'sigma |B|', 'sigma B', 'sigma Bx',
+             'sigma By', 'sigma Bz', 'Proton Temperature',
+             'Proton Density', 'Plasma Flow Speed', 'Plasma Flow Long. Angle',
+             'Plasma Flow Lat. Angle', 'Na/Np', 'Flow Pressure', 'sigma T',
+             'sigma N', 'sigma V', 'sigma phi V', 'sigma theta V',
+             'sigma Na/Np', 'Electric Field', 'Plasma Beta',
+             'Alfven Mach Number', 'Kp', 'R', 'DST Index', 'AE Index',
+             'Proton Flux > 1MeV', 'Proton Flux > 2MeV',
+             'Proton Flux > 4MeV', 'Proton Flux > 10MeV',
+             'Proton Flux > 30MeV',
+             'Proton Flux > 60MeV', 'flag', 'ap index',
+             'f10.7 index', 'PC(N) index', 'AL index (Kyoto)',
+             'AU index (Kyoto)', 'Magnetosonic Mach No.']
+    sfu = u.def_unit('sfu', 10**-22 * u.m**-2 * u.Hz**-1)
+    units = OrderedDict([('Bartels Rotation Number', u.dimensionless_unscaled),
+                         ('ID IMF Spacecraft', u.dimensionless_unscaled),
+                         ('ID SW Plasma Spacecraft', u.dimensionless_unscaled),
+                         ('points(IMF Average)', u.dimensionless_unscaled),
+                         ('points(Plasma Average)', u.dimensionless_unscaled),
+                         ('|B|', u.nT),
+                         ('Magnitude of Avg Field Vector', u.nT),
+                         ('Lat. Angle of Aver. Field Vector', u.deg),
+                         ('Long. Angle of Aver. Field Vector', u.deg),
+                         ('Bx GSE, GSM', u.nT),
+                         ('By GSE', u.nT),
+                         ('Bz GSE', u.nT),
+                         ('By GSM', u.nT),
+                         ('Bz GSM', u.nT),
+                         ('sigma |B|', u.nT),
+                         ('sigma B', u.nT),
+                         ('sigma Bx', u.nT),
+                         ('sigma By', u.nT),
+                         ('sigma Bz', u.nT),
+                         ('Proton Temperature', u.K),
+                         ('Proton Density', u.cm**-3),
+                         ('Plasma Flow Speed', u.km / u.s),
+                         ('Plasma Flow Long. Angle', u.deg),
+                         ('Plasma Flow Lat. Angle', u.deg),
+                         ('Na/Np', u.dimensionless_unscaled),
+                         ('Flow Pressure', u.nPa),
+                         ('sigma T', u.K),
+                         ('sigma N', u.cm**-3),
+                         ('sigma V', u.km / u.s),
+                         ('sigma phi V', u.deg),
+                         ('sigma theta V', u.deg),
+                         ('sigma Na/Np', u.dimensionless_unscaled),
+                         ('Electric Field', u.mV / u.m),
+                         ('Plasma Beta', u.K * u.cm**-3 * u.nT**-2),
+                         ('Alfven Mach Number', u.km * u.cm**-3 / u.nT / u.s),
```

Comment:
This should be dimensionless too

Reply:
I'm pretty sure Alfven Mach Number must be dimensionless, but can you take a look at the README for OMNI? https://cdaweb.gsfc.nasa.gov/pub/data/omni/low_res_omni/omni2.text

Reply:
Their equation is confusing, but this should definitely be dimensionless I think.

Reply:
I checked it out on https://omniweb.gsfc.nasa.gov/form/dx1.html , they are dimensionless, removing them!